src/lib.rs

```diff
@@ -82,8 +90,11 @@ pub use cmp::{Cmp, DefaultCmp};
 pub use compressor::{Compressor, CompressorId};
 pub use db_impl::DB;
 pub use db_iter::DBIterator;
+
+#[cfg(feature = "fs")]
 pub use disk_env::PosixDiskEnv;
-pub use env::Env;
+
+pub use env::{Env, FileLock, RandomAccess, Logger};
```

Comment:
maybe it is better to instead change it to `pub mod env`, as the `FileLock, RandomAccess, Logger` types don't need to appear in the top level of the crate